src-tauri/src/main.rs

```diff
@@ -5,17 +5,24 @@
 mod pipeline;
 use pipeline::pipeline;
 
+use std::env;
+
 // Hardcoded radiance and hdrgen paths for backend testing
-const _FAKE_RADIANCE_PATH: &str = "/usr/local/radiance/bin/";
-const _FAKE_HDRGEN_PATH: &str = "/usr/local/bin/";
+const _FAKE_RADIANCE_PATH: &str = "/usr/local/radiance/bin";
+const _FAKE_HDRGEN_PATH: &str = "/usr/local/bin";
 
 fn main() {
     // === Define hardcoded data for testing ===
 
+    println!("Running fake test!");
```

Comment:
We might want to either remove these print statements or place them within an `if DEBUG` block